oslabs-beta/Kubiquity
# Add Table to Errors

WHAT DOES THIS PR DO? 
1. Adds the react-table library. 
2. Creates a table to display our errors. 
3. Render errors in table. 
4. Minor file name change and location change to the MemoryBarChart component. 

HOW TO TEST THIS PR:
1. Run the following command: `gh pr checkout 10 && npm install && npm run dev` 
2. Verify that the app still starts. 
3. Verify that you no longer see the individual Error components. 
4. Verify that you see a table of errors. 
5. Verify that you can sort the table by clicking the headers. 

ADDITIONAL NOTES: 
1. react-table does not provide a plug-and-play table like our react-bar-chart component, so the table that we have in app is copy and pasted from an example. As a result, it has some default styling and other things that likely will not be used in production. However, I am leaving it in for now, and styling will be covered in a later PR. 
2. I commented out our the mapping and rendering of our Error component in Errors, because we may end up wanting to keep that work. 
3. I left some TODO notes in the ErrorsTable component.
